build.sbt

```diff
@@ -43,3 +55,17 @@ lazy val importer = project
   .in(file("modules/importer"))
   .dependsOn(core, db)
   .settings(commonSettings)
+
+lazy val json = project
+  .in(file("modules/json"))
+  .dependsOn(core)
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+      "io.argonaut"                %% "argonaut"               % argonautVersion,
```

Comment:
Argonaut uses scalaz, circe uses cats. That's the main difference. We may end up switching at some point.